### DISCUSSION TITLE
images and scenes in one gallery

### DISCUSSION THREAD

**Discussion by petrolsniffa123** [stashapp/stash#4690] [Ideas]
this has probably been requested many times but the separation between videos and images in different tabs is such a hinderance to browsing your media, having to constantly switch between the tabs or having to manually add scenes to galleries to show it on the sidepanel for hundreds of images/videos is such a chore. how a new category gets added that just meshes both of them in one gallery

**Comment by Vexz00** [comment]
I 100% agree. Stash could still identify videos and images with the file extensions, even if all files were in one place. I love the "wall" view, the video's previews could seamlessly integrate there.

I have a little workaround, but it's not a complete solution: In the settings I removed "webm" from the scenes extensions and added it to the images extensions. Now all files with the webm extension get added to the images section. I do this because I have many looped clips, for which I don't need the video player's controls. For any longer videos I use the extension "mp4", so the videos get added to the scenes. This way I can use the video player's features. Now to make Stash categorize the files where I want them (scenes/images) I used a bulk renamer tool (in my case KRename) to change the file extensions. Don't worry about changing file extensions, everything still works fine after you change them. But yeah, it would still be much better if Stash would just put everything in one place.
